### PR TITLE
removed comma expressions

### PR DESCRIPTION
**Pull request by nyurik**
Obfuscation contest winner, especially with the "_" as the variable name:
 return arguments.length ? (width = typeof _ === 'function' ? _ : constant(+_), rectangle) : width;

After looking at it for a while, it became clearer, but this is almost impossible to debug
for a non-involved developer.  Also, it clearly shows a usage pattern, which could
be moved to a separate function.

### CODE REVIEW

File: src/util/text.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ canvas(true);
 
 // make dumb, simple estimate if no canvas is available
 function estimateWidth(item) {
-  return fontHeight = height(item), estimate(textValue(item));
+  height(item);
+  return fontHeight = estimate(textValue(item));
```
Comment (jheer) [inline src/util/text.js]: Should be:
```
fontHeight = height(item);
return estimate(textValue(item));
```

File: src/util/text.js [hunk 2]
```diff
@@ -24,7 +25,8 @@ function estimate(text) {
 
 // measure text width if canvas is available
 function measureWidth(item) {
-  return context.font = font(item), measure(textValue(item));
+  font(item);
+  return context.font = measure(textValue(item));
```
Comment (jheer) [inline src/util/text.js]: Should be:
```
context.font = font(item);
return measure(textValue(item));
```

Comment (nyurik) [inline src/util/text.js]: oops, must have been getting tired with some of the rewrites.  Thanks, fixing...

**Comment by jheer** [comment]
Thanks!

**Pull request merged**